engine/swarm/networking.md

```diff
@@ -15,7 +15,7 @@ A Docker swarm generates two different kinds of traffic:
 
 This topic discusses how to manage the application data for your swarm services.
 For more details about swarm networking in general, see the
-[Docker networking reference architecture](https://success.docker.com/Architecture/Docker_Reference_Architecture%3A_Designing_Scalable%2C_Portable_Docker_Container_Networks){: target="_blank" rel="noopener" class="_" }.
+[Docker networking reference architecture](https://docs.docker.com/network/){: target="_blank" rel="noopener" class="_" }.
```

Comment:
As we're now no longer leaving the docs website, I think it's ok to remove the `target=_blank`;

```suggestion
[Docker networking reference architecture](https://docs.docker.com/network/).
```

Reply:
I'm fine with this change, but opening in another tab seems to be a behavior being used by us on other occasions other than when target link is for page outside Docker docs.